packages/webapp/src/components/Forms/Sale/index.jsx

```diff
@@ -205,7 +211,10 @@ const SaleForm = ({
                     value={c.label}
                     hookFormRegister={cropVarietyRegisters[c.value].cropVarietyIdRegister}
                     style={{ marginBottom: '40px' }}
-                    errors={getInputErrors(errors, CROP_VARIETY_VALUES[c.value].CROP_VARIETY_ID)}
+                    errors={getInputErrors(
+                      errors,
+                      cropVarietyRegisterNames[c.value].CROP_VARIETY_ID,
+                    )}
```

Comment:
Did you forget to remove this? lol
You will probably have merge conflict...

Reply:
Yeah probably haha but it is now replaced with the management plan tile